src/app/features/notifications/notifications.component.html

```diff
@@ -0,0 +1,91 @@
+<button
+  mat-icon-button
+  matTooltip="Notifications"
+  i18n-matTooltip="Notifications"
+  [matMenuTriggerFor]="data"
+  (menuOpened)="onNotificationBellClick()"
+>
+  <span
+    [matBadge]="
+      hasUnreadNotificationCount > 0 ? hasUnreadNotificationCount : null
+    "
+    matBadgeColor="accent"
+  >
+    <fa-icon class="white" icon="bell"></fa-icon>
+  </span>
+</button>
+
+<mat-menu #data>
+  <div class="notification-panel">
+    <div
+      class="flex-row justify-space-between align-center notification-panel-header"
+    >
+      <h1 class="notification-title" i18n>Notifications</h1>
+
+      <button
+        mat-icon-button
+        [matMenuTriggerFor]="settingsMenu"
+        (click)="$event.stopPropagation()"
+        icon-button
+      >
+        <fa-icon icon="ellipsis"></fa-icon>
+      </button>
+    </div>
+
+    <!-- Tabs -->
+    <mat-tab-group
+      [(selectedIndex)]="selectedTab"
+      (click)="$event.stopPropagation()"
+    >
+      <mat-tab label="All" i18n-label></mat-tab>
+      <mat-tab label="Unread" i18n-label></mat-tab>
+    </mat-tab-group>
+
+    <ng-container *ngIf="selectedTab === 0">
+      <!-- All Notifications -->
+      <app-notification-item
+        *ngFor="let notification of allNotifications"
+        [notification]="notification"
+        [onMarkAsRead]="markAsRead"
+        [onDeleteNotification]="deleteNotification.bind(this, notification)"
+      ></app-notification-item>
+      <div
+        class="flex-column no-notification-message"
+        *ngIf="allNotifications.length === 0"
+      >
+        <fa-icon class="no-notification-icon" icon="bell"></fa-icon>
+        <span class="no-notification-text" i18n>You have no notifications</span>
+      </div>
+    </ng-container>
+
+    <ng-container *ngIf="selectedTab === 1">
+      <app-notification-item
+        *ngFor="let unreadNotification of unreadNotifications"
+        [notification]="unreadNotification"
+        [onMarkAsRead]="markAsRead"
+        [onDeleteNotification]="
+          deleteNotification.bind(this, unreadNotification)
+        "
+      ></app-notification-item>
+      <div
+        class="flex-column no-notification-message"
+        *ngIf="unreadNotifications.length === 0"
+      >
+        <fa-icon class="no-notification-icon" icon="bell"></fa-icon>
+        <span class="no-notification-text" i18n>You have no notifications</span>
+      </div>
+    </ng-container>
+  </div>
+</mat-menu>
+
+<!-- New mat-menu for ellipsis -->
+<mat-menu #settingsMenu>
+  <button mat-menu-item (click)="markAllRead($event)">
+    <fa-icon icon="bell"></fa-icon>
+    <span class="menu-option" i18n>Mark as notification active</span>
```

Comment:
```suggestion
    <span class="menu-option" i18n>Mark all notifications as read</span>
```

Reply:
The "Mark as notification active" option is used when the user needs to enable notifications for their account. However, to mark all notifications as read in one action, I have added a one more menu item labeled "Mark all as read." When the user clicks on this option, the readStatus of all notifications will be set to true.

Reply:
Ah, I looked at the code to try and understand what you mean by "Mark as notification active". 
So this is something like "Enable notifications"?